addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2027,7 +2027,7 @@ class SnippetsMenu extends Component {
         // blocks should not be taken into account here because they have
         // tooltips with a particular behavior (see _showSnippetTooltip).
         this.tooltips = new Tooltip(this.el, {
-            selector: 'we-title, [title]:not(.oe_snippet)',
+            selector: 'we-title',
```

Comment:
With the change in the selector, shouldn't the `if` be removed: all matches now becoming `we-title` ?
And shouldn't the code that was after the `if` be used instead of `return el.title` in the new tooltips - otherwise we might lose that Firefox-specific fix ?

Reply:
The if should indeed be removed and the Firefox specific code is only for we-title element so we can leave it there.